2023-04-09 21:30:55,692 INFO    MainThread:24537 [wandb_setup.py:_flush():76] Configure stats pid to 24537
2023-04-09 21:30:55,692 INFO    MainThread:24537 [wandb_setup.py:_flush():76] Loading settings from /home/sevi/.config/wandb/settings
2023-04-09 21:30:55,692 INFO    MainThread:24537 [wandb_setup.py:_flush():76] Loading settings from /home/sevi/Documents/GitHub/FYP/wandb/settings
2023-04-09 21:30:55,692 INFO    MainThread:24537 [wandb_setup.py:_flush():76] Loading settings from environment variables: {'silent': 'true'}
2023-04-09 21:30:55,692 INFO    MainThread:24537 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program_relpath': 'train prior.py', 'program': '/home/sevi/Documents/GitHub/FYP/train prior.py'}
2023-04-09 21:30:55,692 INFO    MainThread:24537 [wandb_init.py:_log_setup():461] Logging user logs to /home/sevi/Documents/GitHub/FYP/wandb/run-20230409_213055-dool58nt/logs/debug.log
2023-04-09 21:30:55,692 INFO    MainThread:24537 [wandb_init.py:_log_setup():462] Logging internal logs to /home/sevi/Documents/GitHub/FYP/wandb/run-20230409_213055-dool58nt/logs/debug-internal.log
2023-04-09 21:30:55,693 INFO    MainThread:24537 [wandb_init.py:init():495] calling init triggers
2023-04-09 21:30:55,693 INFO    MainThread:24537 [wandb_init.py:init():498] wandb.init called with sweep_config: {}
config: {'prior': {'clip': {'make': 'openai', 'model': 'ViT-B/32', 'base_model_kwargs': None}, 'net': {'dim': 512, 'depth': 12, 'max_text_len': 77, 'num_timesteps': 1000, 'num_time_embeds': 1, 'num_image_embeds': 1, 'num_text_embeds': 1, 'dim_head': 64, 'heads': 12, 'ff_mult': 4, 'norm_in': False, 'norm_out': True, 'attn_dropout': 0.05, 'ff_dropout': 0.05, 'final_proj': True, 'normformer': True, 'rotary_emb': True}, 'image_embed_dim': 512, 'image_size': 224, 'image_channels': 3, 'timesteps': 1000, 'sample_timesteps': 64, 'cond_drop_prob': 0.1, 'loss_type': 'l2', 'predict_x_start': True, 'beta_schedule': 'cosine', 'condition_on_text_encodings': True}, 'data': {'image_url': '/home/sevi/datasets/example/examples/{00000..00037}.tar', 'meta_url': '/home/sevi/datasets/example/example output/metadata', 'splits': {'train': 0.8, 'val': 0.1, 'test': 0.1}, 'batch_size': 128, 'num_data_points': 100000, 'eval_every_seconds': 1600}, 'train': {'epochs': 5, 'lr': 0.00011, 'wd': 0.0602, 'max_grad_norm': 0.5, 'use_ema': True, 'ema_beta': 0.9999, 'amp': False, 'warmup_steps': 50, 'save_every_seconds': 3600, 'eval_timesteps': [64, 1000], 'best_validation_loss': 1000000000.0, 'current_epoch': 0, 'num_samples_seen': 0, 'random_seed': 84513}, 'tracker': {'data_path': '.prior', 'overwrite_data_path': True, 'log': {'log_type': 'wandb', 'resume': False, 'auto_resume': False, 'verbose': True, 'wandb_resume': False, 'wandb_entity': 'key=b8a9408d1abc8463d78713f047bf787d23bf38e8', 'wandb_project': 'prior_debugging'}, 'load': None, 'save': [{'save_to': 'local', 'save_all': False, 'save_latest': True, 'save_best': True, 'save_latest_to': 'latest_checkpoint.pth', 'save_best_to': 'best_checkpoint.pth', 'save_type': 'checkpoint'}]}, 'Distributed': False, 'DistributedType': <DistributedType.NO: 'NO'>, 'NumProcesses': 1, 'MixedPrecision': 'no'}
2023-04-09 21:30:55,693 INFO    MainThread:24537 [wandb_init.py:init():548] starting backend
2023-04-09 21:30:55,693 INFO    MainThread:24537 [backend.py:_multiprocessing_setup():97] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2023-04-09 21:30:55,709 INFO    MainThread:24537 [backend.py:ensure_launched():217] starting backend process...
2023-04-09 21:30:55,722 INFO    MainThread:24537 [backend.py:ensure_launched():222] started backend process with pid: 24556
2023-04-09 21:30:55,723 INFO    MainThread:24537 [wandb_init.py:init():558] backend started and connected
2023-04-09 21:30:55,728 INFO    MainThread:24537 [wandb_init.py:init():634] updated telemetry
2023-04-09 21:30:55,760 INFO    MainThread:24537 [wandb_init.py:init():665] communicating run to backend with 30 second timeout
2023-04-09 21:31:25,760 INFO    MainThread:24537 [interface_shared.py:_communicate_run():326] couldn't get run from backend
2023-04-09 21:31:25,760 ERROR   MainThread:24537 [wandb_init.py:init():674] backend process timed out
2023-04-09 21:31:25,761 ERROR   MainThread:24537 [wandb_init.py:init():685] encountered error: Error communicating with wandb process
try: wandb.init(settings=wandb.Settings(start_method='fork'))
or:  wandb.init(settings=wandb.Settings(start_method='thread'))
For more info see: https://docs.wandb.ai/library/init#init-start-error
2023-04-09 21:31:25,766 ERROR   MainThread:24537 [wandb_init.py:init():1070] error
Traceback (most recent call last):
  File "/home/sevi/anaconda3/envs/AdvAI/lib/python3.8/site-packages/wandb/sdk/wandb_init.py", line 1043, in init
    run = wi.init()
  File "/home/sevi/anaconda3/envs/AdvAI/lib/python3.8/site-packages/wandb/sdk/wandb_init.py", line 689, in init
    backend.cleanup()
  File "/home/sevi/anaconda3/envs/AdvAI/lib/python3.8/site-packages/wandb/sdk/backend/backend.py", line 246, in cleanup
    self.interface.join()
  File "/home/sevi/anaconda3/envs/AdvAI/lib/python3.8/site-packages/wandb/sdk/interface/interface_shared.py", line 475, in join
    super().join()
  File "/home/sevi/anaconda3/envs/AdvAI/lib/python3.8/site-packages/wandb/sdk/interface/interface.py", line 666, in join
    _ = self._communicate_shutdown()
  File "/home/sevi/anaconda3/envs/AdvAI/lib/python3.8/site-packages/wandb/sdk/interface/interface_shared.py", line 472, in _communicate_shutdown
    _ = self._communicate(record)
  File "/home/sevi/anaconda3/envs/AdvAI/lib/python3.8/site-packages/wandb/sdk/interface/interface_shared.py", line 226, in _communicate
    return self._communicate_async(rec, local=local).get(timeout=timeout)
  File "/home/sevi/anaconda3/envs/AdvAI/lib/python3.8/site-packages/wandb/sdk/interface/interface_shared.py", line 231, in _communicate_async
    raise Exception("The wandb backend process has shutdown")
Exception: The wandb backend process has shutdown
